html/browsers/browsing-the-web/unloading-documents/unload/006-1.html

```diff
@@ -4,9 +4,12 @@
 onpagehide = function() {
   onpagehide = null;
   setTimeout(function() {
-               parent.t.done()
+               parent.t.step_func(function() { assert_unreached('setTimeout survived navigatoin'); })
```

Comment:
`parent.t.unreached_func('setTimeout survived navigatoin')` would be more compact.

Reply:
done